diff --git a/node_modules/ssb-keys/storage.js b/node_modules/ssb-keys/storage.js
index 9f628c6..6c4d233 100644
--- a/node_modules/ssb-keys/storage.js
+++ b/node_modules/ssb-keys/storage.js
@@ -100,7 +100,7 @@
     var keyfile = constructKeys(keys, legacy)
     mkdirp(path.dirname(filename), function (err) {
       if(err) return cb(err)
-      fs.writeFile(filename, keyfile, {mode: 0400}, function(err) {
+      fs.writeFile(filename, keyfile, {mode: 400}, function(err) {
         if (err) return cb(err)
         cb(null, keys)
       })
@@ -112,7 +112,7 @@
     var keys = generate(curve)
     var keyfile = constructKeys(keys, legacy)
     mkdirp.sync(path.dirname(filename))
-    fs.writeFileSync(filename, keyfile, {mode: 0400})
+    fs.writeFileSync(filename, keyfile, {mode: 400})
     return keys
   }